src/cmds/wallet.py

```diff
@@ -95,33 +33,21 @@ def print_transaction(tx: TransactionRecord, verbose: bool):
         print("")
 
 
-async def get_transaction(args, wallet_client, fingerprint: int):
-    if args.id is None:
```

Comment:
This checks should be done at command click command level.

Reply:
makes sense